File: Atividade2/FuncionarioJ/FuncionarioJ/Gerente.java
package FuncionarioJ;
public class Gerente extends Funcionario{

    public Gerente(String nome, int id, double salario_base){
        super(nome, id, salario_base);
    }

    @Override
    public double calcularBonus(){
        double salario_base = getSalarioBase();

        if(salario_base > 10000 && salario_base <= 50000) return salario_base * 0.20;
        if (salario_base > 50000) return salario_base * 0.25;
        return 0;
    }
}

File: Atividade2/FuncionarioJ/FuncionarioP/Gerente.py
from Funcionario import Funcionario

class Gerente(Funcionario):
    def __init__(self, nome, id, salario_base):
        super().__init__(nome, id, salario_base)

    def calcular_bonus(self) -> float:
        if self.salario_base > 10000 and self.salario_base <= 50000:
            return self.salario_base * 0.10
        elif self.salario_base <= 10000:
            return self.salario_base * 0.15
        else:
            return 0
        